components/Form.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+import { useState } from "react";
+import CharacterCounter from "./CharacterCounter";
+
+export default function Form({ handleSubmit }) {
+  const [count, setCount] = useState(0);
+  const [countDescription, setCountDescription] = useState(0);
+
+  return (
+    <StyledForm onSubmit={handleSubmit}>
+      <label htmlFor="name">Name:</label>
+      <input
+        id="name"
+        name="name"
+        type="text"
+        pattern="^[a-zA-Z0-9äüöÄÜÖ][a-zA-Z0-9-_ äüöÄÜÖß.] {1,}"
```

Comment:
Hi Marc, somehow I have the feeling that this pattern is preventing that I can type any Name. I checked the preview just to remember what your app is about and I can type any Name.

Reply:
Hey niklas, i think the problem was the space after the second []

Reply:
Hey Marc, it works like it is supposed to. 
Just one note: 
if you use the English language you should allow apostrophes (') in your pattern.

Reply:
Thank you, the pattern is still being improved. I will implement this in the next pr 🙂